src/examples/correctExample.json

```diff
@@ -0,0 +1,51 @@
+{
+  "nodeType": "formulas",
+  "forest": [
+    {
+      "nodeType": "formulas",
+      "forest": [
+        {
+          "nodeType": "formulas",
+          "forest": [
+            {
+              "nodeType": "formulas",
+              "forest": [],
+              "formulas": [
+                {
+                  "resolved": false,
+                  "row": 3,
+                  "value": "~E"
+                }
+              ],
+              "rule": "A"
+            }
+          ],
+          "formulas": [
+            {
+              "resolved": false,
+              "row": 2,
+              "value": "(~B\\/C)\\/E"
+            }
+          ],
+          "rule": "A"
+        }
+      ],
+      "formulas": [
+        {
+          "resolved": false,
+          "row": 1,
+          "value": "~(C\\/D)"
+        }
+      ],
+      "rule": "A"
+    }
+  ],
+  "formulas": [
+    {
+      "resolved": false,
+      "row": 0,
+      "value": "A&B"
+    }
+  ],
+  "rule": "A"
```

Comment:
There should only be one node with rule "A".
Put all the other assumptions in this "formulas" array.